features/support/env.rb

```diff
@@ -94,5 +95,11 @@
   create(:corporate_partner)
 end
 
+Before('@auth-required') do
+  allow(Authenticable).to receive(:username)  { ENV['AUTH_USERNAME'] }
+  allow(Authenticable).to receive(:password)  { ENV['AUTH_PASSWORD'] }
```

Comment:
Oh! Will these methods (username, password) on Authenticable not return the value of these env vars when when testing? What I mean is, it looks like these stubs reimplement the same thing as the real module does, is that not the case?